client/src/sections/resources/Cartography.tsx

```diff
@@ -0,0 +1,168 @@
+import Line from "../../components/Line";
+import Tile from "../../components/Tile";
+import { useMyLevel } from "../../hooks/useCall";
+import useGet from "../../hooks/useGet";
+import useHave from "../../hooks/useHave";
+import { $skill, $item } from "../../util/makeValue";
+import { plural } from "../../util/text";
+
+// ==== TILE TO-DO LIST ==================
+//   - Want to figure out a good way to make the bullets into links to the respective zones or containers.
+//   - Add more map targets with more conditions on them. (If you have a spit up, you probably want to map a good spit dude, for instance.)
+//   - Better ways to re-order & re-prioritize the map targets.
+//   - Can we do a for loop within the <Tile> to enumerate mapTarget? That would be so much better if it is at all possible.
+
+/**
+ * Class used to store map targets. Unless otherwise specified, all params are strings.
+ * @param monster The monster in question
+ * @param zone The zone you'll find the monster in
+ * @param level The level this map target is feasible (number)
+ * @param access A way to pass in a custom condition for accessibility (boolean)
+ */
+
+class mapTarget {
+  monster: string;
+  zone: string;
+  level: number;
+  access: boolean;
+  turnsTilGROPs: number;
+
+  constructor(
+    monster: string,
+    zone: string,
+    level: number,
+    access?: boolean,
+    turnsTilGROPs?: number
+  ) {
+    this.monster = monster;
+    this.zone = zone;
+    this.level = level;
+    this.access = access ?? true;
+    this.turnsTilGROPs = turnsTilGROPs ?? 0;
+  }
+
+  /**
+   * Returns true/false depending on if you are at a level high enough to access this monster.
+   * @param monster The monster in question
+   */
+  accessible(userLevel: number): boolean {
+    return this.access && userLevel > this.level;
+  }
+
+  /**
+   * Returns a string for the mapTarget to feed into your <Line> statement.
+   */
+  formatString(): string {
```

Comment:
This should just return a `ReactNode` that you can plug directly into your tile.